include/mbedtls/ssl.h

```diff
@@ -45,41 +45,63 @@
  * MBEDTLS_ECDSA_C which does not seem correct.
  */
 #if defined(MBEDTLS_ECDH_C) || defined(MBEDTLS_ECDSA_C)
-#include "mbedtls/ecdh.h"
+#    include "mbedtls/ecdh.h"
 #endif
 
 #if defined(MBEDTLS_HAVE_TIME)
-#include "mbedtls/platform_time.h"
+#    include "mbedtls/platform_time.h"
 #endif
 
 #if defined(MBEDTLS_USE_PSA_CRYPTO)
-#include "psa/crypto.h"
+#    include "psa/crypto.h"
 #endif /* MBEDTLS_USE_PSA_CRYPTO */
 
 /*
  * SSL Error codes
  */
-#define MBEDTLS_ERR_SSL_CRYPTO_IN_PROGRESS                -0x7000  /**< A cryptographic operation is in progress. Try again later. */
-#define MBEDTLS_ERR_SSL_FEATURE_UNAVAILABLE               -0x7080  /**< The requested feature is not available. */
-#define MBEDTLS_ERR_SSL_BAD_INPUT_DATA                    -0x7100  /**< Bad input parameters to function. */
-#define MBEDTLS_ERR_SSL_INVALID_MAC                       -0x7180  /**< Verification of the message MAC failed. */
-#define MBEDTLS_ERR_SSL_INVALID_RECORD                    -0x7200  /**< An invalid SSL record was received. */
-#define MBEDTLS_ERR_SSL_CONN_EOF                          -0x7280  /**< The connection indicated an EOF. */
-#define MBEDTLS_ERR_SSL_DECODE_ERROR                      -0x7300  /**< A message could not be parsed due to a syntactic error. */
+#define MBEDTLS_ERR_SSL_CRYPTO_IN_PROGRESS \
```

Comment:
Even with penalty `PenaltyBreakComment: 0` these lines are split.
The solution here is to either modify scripts/generate_errors.pl to accept line continuation or add `clang-format-off/on` comments around `MBEDTLS_ERR_` defines.

Reply:
Another possibility is to rewrite them manually before to apply the reformatting tool. For example:
```C
/**< A cryptographic operation is in progress. Try again later. */
#define MBEDTLS_ERR_SSL_CRYPTO_IN_PROGRESS                -0x7000
```

Reply:
I added clang format guards. Please check if you're ok with it

Reply:
I would prefer to fix this manually rather than introducing clang format guards.

Reply:
I'd prefer to lift the requirement of having the comment on the same line: it forces those lines to be very long. But that requires improvin `generate_errors.pl`.

Reply:
I guess that this formatting
```
/**< A cryptographic operation is in progress. Try again later. */
#define MBEDTLS_ERR_SSL_CRYPTO_IN_PROGRESS                -0x7000
```
will have to be applied manually.

Reply:
Or maybe not manually.. https://regex101.com/r/Pyr0nT/1

Reply:
Transforming the source code is a simple perl one-liner. Updating the consumer script is more than a one-liner.